### DISCUSSION TITLE
Search result Pagination - does not take user to top of new page

### DISCUSSION THREAD

**Discussion by stuartcporter** [microsoft-search/pnp-modern-search#1917] [Ideas]
**Version used**
4.5.3

**Describe the bug**
on a custom search page - using search results web part set to show 10 items per page and paging enabled.

**To Reproduce**
1. search for item with more than 10 results 
2. Click on page 2 beneath the results
3. notice page refreshes and loads the second page of results - however the user is left at the bottom of the page - unlike regular sharepoint with refreshes and takes you to the top of the second page

**Expected behavior**
new page of results to load and user is taken to top of page

**Desktop (please complete the following information):**
 - Browser 
 - Edge, Chrome

**Comment by FranckyC** [comment]
Moved it to discussions as enhancement

**Reply by ferrarirosso** [reply]
Hi @FranckyC 
I am interested in solving/coding this enhancement idea.
Should I open an issue and pull my code ?
Thx

**Reply by ferrarirosso** [reply]
Follow-Up
Any interest in discussing implementation of this feature ?
If not, we will custom implement it

**Reply by wobba** [reply]
Feel free to PR a solution.

**Reply by ferrarirosso** [reply]
I just found that a solution was already pushed [Scroll to view top results](https://github.com/microsoft-search/pnp-modern-search/pull/2244)
I am not sure why [scrollIntoViewIfNeeded](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoViewIfNeeded) is used, because this method is marked as proprietary
The correct method [scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) is used as fallback, instead of the not supported one.
So the code executes always scrollIntoViewIfNeeded instead of scrollIntoView

**Comment by kasperbolarsen** [comment]
tested in v4.18.2 and it works fine now
